docs/articles/tasks.md

```diff
@@ -0,0 +1,35 @@
+# Tasks
+
+### What is a Task?
+
+Task is a process which is launched by Mesos executor on the Mesos Agent. When Marathon launches an instance of the application definition, Marathon sends a command to Mesos to launch one or more tasks.
+
+### How can I use a Task?
+Tasks are automatically created by Mesos once application definition is being launched.
```

Comment:
> Tasks are automatically created by Mesos

What does "automatically" specifically means? It feels like the description above still doesn't quite explain what a task is. How about an example e.g .`"cmd": "sleep 100"`, describing the process of starting it on some agent (going through offer matching, executor etc.), throwing in a few points about isolation (with links to Mesos docs) and describing the fact, that user can see tasks sandbox (stderr, stdout)...